### ISSUE TITLE
Cannot create a docker image for coreutils

### ISSUE THREAD

**Issue by agb94** [Suresoft-GLaDOS/defects4cpp#61]
Hi,
whenever I try to build an image for `coreutils`, I got the following error message:

```
[coreutils] running test suites (coverage)
  Starting container
Creating a new docker image for coreutils
The command '/bin/sh -c apt update   apt install -y autoconf automake bison gettext git gperf gzip help2man m4 make perl tar texinfo wget xz-utils gcc autopoint rsync python3 python3-pip' returned a non-zero code: 100
Could not build image. Please open an issue at https://github.com/Suresoft-GLaDOS/defects4cpp/issues
```

**Comment by HansolChoe** [comment]
`&&` was missing `apt update` and `apt install` in Dockerfile.

This bug is fixed at ([e6f7e06](https://github.com/Suresoft-GLaDOS/defects4cpp/commit/e6f7e06baca3be7c236d700adf5c297689f790f1)).

Please checkout and build `coreutils` again, and let me know this fix works.

By the way, thank you for your report!

**Comment by HansolChoe** [comment]
Close this issue.
Please reopen this issue if the fix not works.
